immersion/kubeflow_pipelines/cicd/solutions/lab-03_vertex.ipynb

```diff
@@ -0,0 +1,238 @@
+{
```

Comment:
<p>nit suggestion: you could turn the command to a link to https://cloud.google.com/sdk/gcloud/reference/builds/submit</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/mlops-on-gcp/pull/117/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Done!</p>